src/main/java/it/pagopa/transactions/exceptions/JWTTokenGenerationException.java

```diff
@@ -0,0 +1,8 @@
+package it.pagopa.transactions.exceptions;
+
+public class JWTTokenGenerationException extends Exception {
```

Comment:
If this is not meant to be used as a checked exception we should extend `RuntimeException`

Reply:
Fixed into https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/120/commits/b3550051b8e712d78f388ea21c8035cecf04fc14